packages/react-hooks/src/BasicMessageProvider.tsx

```diff
@@ -28,6 +29,7 @@ export const useBasicMessagesByConnectionId = (connectionId: string): BasicMessa
 
 interface Props {
   agent: Agent | undefined
+  children: React.ReactNode
 }
```

Comment:
This is fine, but just so you know. You can do `const Component = React.FC<PropsWithChildren<Props>>` and then you don't have to declare the type for children yourself

Reply:
> It seems CI is failing

Seems to be failing with the redux extension for some reason